include/my_sdnotify.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef _my_sdnotify_h
+#define _my_sdnotify_h
+
+#ifdef HAVE_SYSTEMD
+#include <systemd/sd-daemon.h>
+
+#else
+
+#define sd_listen_fds(X) 0
+
+#define sd_notify(X, Y, Z)
+static int sd_notifyf(int unset_environment, const char *format, ...) {}
+
+#endif
+
+#endif /* _my_sdnotify_h */
```

Comment:
This script doesn't do just sd_notify, but also some other systemd specific stuff. So name my_sdnotify.h doesn't seem to be applicable anymore. May be rename to something like my_systemd.h?